packages/tempus/test/Integrator/Tempus_IntegratorBasic_ref.xml

```diff
@@ -8,6 +8,7 @@
       <Parameter docString="" id="4" isDefault="true" isUsed="true" name="Storage Limit" type="int" value="2"/>
       <ParameterList name="Interpolator">
         <Parameter name="Interpolator Type" type="string" value="Lagrange"/>
+        <Parameter name="Order" type="int" value="0"/>
```

Comment:
The order is now required?

Reply:
No, but to check if the defaults have been set correctly, I needed to have it set in the reference ParameterList.